iceoryx_posh/source/roudi/roudi_cmd_line_parser.cpp

```diff
@@ -52,11 +53,20 @@ void CmdLineParser::parse(int argc, char* argv[], const CmdLineArgumentParsingMo
             std::cout << "-l, --log-level <LEVEL>           Set log level." << std::endl;
             std::cout << "                                  <LEVEL> {off, fatal, error, warning, info, debug, verbose}"
                       << std::endl;
+            std::cout << "-c, --compatibility               Set compatibility check level between runtime and RouDi"
+                      << std::endl;
+            std::cout << "                                  off: no check" << std::endl;
+            std::cout << "                                  major: same major version " << std::endl;
+            std::cout << "                                  minor: same minor version + major check" << std::endl;
+            std::cout << "                                  patch: same patch version + minor check" << std::endl;
+            std::cout << "                                  commitId: same commit ID + patch check" << std::endl;
```

Comment:
Does anybody know if there is a convention on how to name cmd line arguments? Like having a hyphen? At least we already have `--log-level` and `--monitoring-mode`.

Reply:
http://docopt.org/